exalearn-ui/src/app/services/audio-cloud.service.ts

```diff
@@ -7,29 +7,20 @@ import { finalize } from 'rxjs/operators';
 	providedIn: 'root'
 })
 export class AudioCloudService {
-	/* 
-		There is Aleksey Olshevsky code.
-		He uses it for his audition component.
-		I'll remove it when we will receive data from the server 
-	*/
-
-	files: any = [
-		{
-			url: 'https://mp3.chillhop.com/serve.php/?mp3=10536',
-			name: "She Won't Say",
-			artist: 'Psalm Trees, Guillaume Muschalle'
-		}
-	];
-
-	getFiles() {
-		return of(this.files);
-	}
-
 	private storageRef: AngularFireStorageReference;
 	private uploadFile: AngularFireUploadTask;
+	files: any;
```

Comment:
Do we really not now which type is file going to be?

Reply:
Created new type, thx